Tests/StreamChatTests/Controllers/MessageController/MessageController_Tests.swift

```diff
@@ -410,7 +410,7 @@ final class MessageController_Tests: XCTestCase {
 
         // Check the order of replies is correct
         let bottomToTopIds = [reply1, reply2].sorted { $0.createdAt < $1.createdAt }.map(\.id)
-        XCTAssertEqual(controller.replies.map(\.id), bottomToTopIds)
+        AssertAsync.willBeEqual(controller.replies.map(\.id), bottomToTopIds)
```

Comment:
we're actually trying to get rid of `AssertAsync`, best to use raw expectations here

Reply:
I'll try to change these